Sort and copyedit changelog. (#8933)
ethereum/solidity: Changelog.md

### 0.6.8 (2020-05-14)

Important Bugfixes:
 * Add missing callvalue check to the creation code of a contract that does not define a constructor but has a base that does define a constructor.

>  * Disallow index range (array slice) accesses for arrays with dynamically encoded base types.
>  * Fixed that string literals containing backslash characters could cause incorrect code to be generated when passed directly to function calls or encoding functions when ABIEncoderV2 is active.

Reviewer: ```suggestion
 * Disallow array slices stemming from index range accesses to arrays with dynamically encoded base types.
 * String literals containing backslash characters can no longer cause incorrect code to be generated when passed directly to function calls or encoding functions when ABIEncoderV2 is active.
```
Maybe, if we're changing them anyways :-)?

Or for the first one:
``* Disallow array slices of arrays with dynamically encoded base types.``

Language Features:

>  * Implemented ``type(X).min`` and ``type(X).max`` for every integer type ``X`` that returns the smallest and largest value representable by the type.

Reviewer 2: ```suggestion
 * Implemented ``type(T).min`` and ``type(T).max`` for every integer type ``T`` that returns the smallest and largest value representable by the type.
```

Language Features:
 * Implemented ``type(X).min`` and ``type(X).max`` for every integer type ``X`` that returns the smallest and largest value representable by the type.

Compiler Features:

>  * Commandline Interface: Don't ignore `--yul-optimizations` in assembly mode.

Reviewer 2: Why remove this?

Reviewer: Wow, I just overlooked that!

Author: merge problem I think
